sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosMicrometerMeterOptions.java

```diff
@@ -18,6 +18,8 @@ public final class CosmosMicrometerMeterOptions {
     private EnumSet<TagName> suppressedTagNames;
     private Boolean isEnabled;
 
+    private Boolean applyDiagnosticThresholds;
```

Comment:
I wonder why other types are object types and not primitives? is there any value in setting them to null?
If not, we should change them as well?

Reply:
No - intentionally not using primitive type because null has a meaning - if not set it will inherit client-wide settings - if set this will override the defaults set on the client (per meter)